lib/keyResolver.js

```diff
@@ -0,0 +1,20 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const axios = require('axios');
+const brHttpsAgent = require('bedrock-https-agent');
+
+module.exports = keyResolver;
+
+// FIXME: make more restrictive, support `did:key` and `did:v1`
```

Comment:
The other way to make this more restrictive is based on a config option that specifies where the KMS is.

Reply:
Added this a TODO.